### DISCUSSION TITLE
PDF view instead of downloading it

### DISCUSSION THREAD

**Discussion by TimKadyrov** [MicrosoftEdge/WebView2Feedback#3880] [Q&A]
It may be related to #1127 but I'm not sure since I want PDF to be displayed.

When I pass URL to PDF file, wv2 first downloads it and then user need to select what to do with it.
If selection is made to open it in viewer - it will be opened in a new wv2 window, and not in the one embedded in application.

Is it possible to control somehow that downloaded PDF is automatically opened in wv2 control which received this download link?

Very much appreciated.

**Comment by hunkydoryrepair** [comment]
do you control the server? This can be controlled by `Content-Disposition` header, I believe.

You could also try using Javascript, something like

`ExecuteScriptAsync("window.location = '/path-to/my.pdf'")`
